src/amo/components/RatingManagerNotice/styles.scss

```diff
@@ -0,0 +1,10 @@
+@import '~amo/css/styles';
+
+.RatingManagerNotice-savedRating {
+  color: $grey-50;
```

Comment:
For some reason this is overriding `$green-90` which should be the text color for success notices.

<img width="352" alt="screenshot 2018-09-12 16 01 59" src="https://user-images.githubusercontent.com/55398/45453030-33388000-b6a5-11e8-83e0-d2a276b2d6d3.png">

Reply:
I'm not seeing this. I.e., I see the green message as you've shown above. But that doesn't really matter because we no longer need to request this grey colour for the component, so I've removed that style.